src/pods/employee/components/data.validations.ts

```diff
@@ -0,0 +1,25 @@
+import { ValidationSchema, Validators } from '@lemoncode/fonk';
+import { createFormikValidation } from '@lemoncode/fonk-formik';
+import { matchField } from '@lemoncode/fonk-match-field-validator';
+
+export const validationSchema: ValidationSchema = {
+  field: {
+    name: [Validators.required],
+    email: [
+      {
+        validator: Validators.email,
+      },
+      {
+        validator: Validators.required,
+      },
+    ],
+    temporalPassword: [
+      {
+        validator: matchField.validator,
+        customArgs: { field: 'password' },
+      },
```

Comment:
This one is strange, _matchfield_ compare two fields, is ok for reset password scenarios (you have to reenter the info in a new field to double check), in this case we don't have that.

The temporal password must be required only if it's in insert mode.

https://github.com/Lemoncode/fonk-required-by-field-validator

To know if you are in insert mode you can check that 'id' field is empty string